pytorch/xpu-requirements.txt

```diff
@@ -1,7 +1,8 @@
-torch==2.1.0.post2+cxx11.abi
-torchvision==0.16.0.post2+cxx11.abi
-torchaudio==2.1.0.post2+cxx11.abi
-intel_extension_for_pytorch==2.1.30+xpu
-oneccl_bind_pt==2.1.300+xpu
+torch==2.1.0.post3+cxx11.abi
+torchvision==0.16.0.post3+cxx11.abi
+torchaudio==2.1.0.post3+cxx11.abi
+intel_extension_for_pytorch==2.1.40+xpu
+oneccl_bind_pt==2.1.400+xpu
 --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us
-setuptools==71.1.0
+setuptools==69.5.1
```

Comment:
This version of setuptools has a critical CVE

Reply:
@tylertitsworth  that version of setuptools is required for successful ipex import on xpu. The ipex team is made aware of this and the other vulnerability from torch